<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdate</class>
 <widget class="QWidget" name="AutoUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_MaIP">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>185</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MaIP">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>186</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addMa">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>185</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
